service/resources/submission.py

```diff
@@ -0,0 +1,30 @@
+import json
+import jsend
+import falcon
+from sqlalchemy.ext.declarative import declarative_base
+import sqlalchemy as sa
+from sqlalchemy.sql import func
+
+# from pprint import pprint
+
+Base = declarative_base()
+
+class Submission(Base):
+    __tablename__ = 'submission'
+    id = sa.Column('id', sa.Integer, primary_key=True)
+    data = sa.Column('data', sa.Text, nullable=False)
```

Comment:
should we enforce the data column to be `JSON` type versus generic `Text` type?

Reply:
i chose to go with text in case there is pii and we need to encrypt.